tests/test_endpoints.py

```diff
@@ -134,6 +134,9 @@ def test_pass_thru():
 
 def test_status_interface(config):
 
+    # We have this FakeSocket, so we can emulate the gunicorn.socket
+    # environment variable. Is it gunicorn specif and represents a
+    # socket object.
```

Comment:
"Is it" sounds like a question, and "specif" should probably be "specific". Please reword?